README.md

```diff
@@ -225,13 +225,29 @@ This specification adds new features and includes all parts from 3.0.
 
 The package should include `getDate` function to expose the underlying date value. For coordination numbers the day should be removed with 60 days to get the correct day for a real date.
 
-## T-Number
+## Interim-Number
 
-The package should support T-numbers `YYMMDD-TXXX` and include `isInterimNumber` function in the same way as `isCoordinationNumber`
+_Also called T-Number_
 
-The `format` function should respect T-number and format the number both in the long and short way with the letter `T` in the 4-digit number.
+The package should support Interim-numbers `YYMMDD-Xnnn` (where X is a letter rather than number) and include `isInterimNumber` function in the same way as `isCoordinationNumber`
+
+The `format` function should respect Interim-number and format the number both in the long and short way with the letter at the first of the four in the 4-digit number.
+
+Read more about interim-number at [KTH](https://www.kth.se/en/student/studier/living-in-sweden/swedish-personal-identification-number/swedish-personal-identification-number-1.443883) and at [sunet](https://wiki.sunet.se/display/SWAMID/Svenska+personnummer%3A+norEduPersonNIN%2C+personalIdentityNumber+och+schacDateOfBirth).
+
+To make the package future-proof, we support all 11 interim letters: `T, R, S, U, W, X, J, K, L, M, N` where all are replaced with a `1` in the lun calculation.
```

Comment:
We should add `P` to the list since SU seems to use it:
* http://www.su.se/polopoly_fs/1.1930.1276934061!/P%20and%20T%20number.pdf (page 2)
* https://www.mira.se/knowledge-base/vad-heter-personnummer-och-samordningsnummer-pa-engelska/

Reply:
`lun` => `luhn`